app/views/signs/show.html.haml

```diff
@@ -16,10 +16,15 @@
       .glosses-container.glosses
         %h2.main_gloss= @sign.gloss_main
         %h2.main_gloss.maori-gloss= @sign.gloss_maori
-      = link_to "#", { class: "card__button add-to-vocab-btn card__button--peach", "data-sign-id": @sign.id } do
-        %span.icon-container
-          %i.fi-plus
-        %span.text-container Add to Vocab Sheet
+      .button-container
+        = link_to "/images/signs/400-400/#{@sign.drawing}", { "data-sign-id": @sign.id } do
```

Comment:
the path `/images/signs/400-400` is used a few times. Can it be in a variable or helper, some other way to DRY so it's not repeated.

Reply:
something smart and short link `link_to @sign.image` or `sign_image_path(@sign)`